connectors/tests/fake_sources.py

```diff
@@ -57,6 +58,11 @@ async def validate_filtering(cls, filtering):
             state=FilteringValidationState.VALID, errors=[]
         )
 
+    async def validate_configuration(self):
+        if self.configuration_invalid:
+            raise ValueError("I fail when validating configuration")
+        pass
```

Comment:
remove this `pass` (only to be used when a function is empty)

Reply:
Ahhh, missed it - removed at https://github.com/elastic/connectors-python/pull/552/commits/dba6430863e5abd7de106cc113e6b9ba75c40d46